tasks/update-ocp-tag/update-ocp-tag.yaml

```diff
@@ -47,15 +47,33 @@ spec:
 
         # Function to replace tag in an image
         replace_tag() {
-            local updatedImage="${1%:*}:$(params.ocpVersion)"
-            echo "$updatedImage"
+            # replaces the tag if the placeholder exists
+            echo "${1}" |sed -E "s/\{\{(\\s+)?OCP_VERSION(\\s+)?\}\}/$(params.ocpVersion)/g"
+        }
+
+        # validateOCPVersion function validates if the base image ocp ver matches with the one in the tested index
```

Comment:
It would be nice to specify the expected args of the function.

Reply:
sure, done.